examples/with_transaction.php

```diff
@@ -0,0 +1,56 @@
+<?php
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use MongoDB\Client;
+use MongoDB\Driver\Session;
+
+use function assert;
+use function dirname;
+use function getenv;
+use function is_object;
+use function MongoDB\BSON\fromPHP;
+use function MongoDB\BSON\toRelaxedExtendedJSON;
+use function MongoDB\with_transaction;
+use function printf;
+
+require dirname(__FILE__) . '/../vendor/autoload.php';
+
+function toJSON(object $document): string
+{
+    return toRelaxedExtendedJSON(fromPHP($document));
+}
+
+$client = new Client(getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/');
```

Comment:
Would it be worth adding a comment here and in `changestream.php` to note that both features require replication? Maybe that's not necessary as the majority of users will likely just read/copy the code instead of execute them directly.

Reply:
Good point, done.